Disable various compiler configs (currently warning suppression). Like  chromium-102-compiler.patch but for node

--- src/third_party/electron_node/BUILD.gn.old
+++ src/third_party/electron_node/BUILD.gn
@@ -142,12 +142,6 @@ config("node_lib_config") {
 config("node_lib_config") {
   include_dirs = [ "src" ]
 
-  cflags = [
-    "-Wno-shadow",
-    # FIXME(deepak1556): include paths should be corrected,
-    # refer https://docs.google.com/presentation/d/1oxNHaVjA9Gn_rTzX6HIpJHP7nXRua_0URXxxJ3oYRq0/edit#slide=id.g71ecd450e_2_702
-    "-Wno-microsoft-include",
-  ]
 
   configs = [ ":node_features" ]
 
@@ -250,17 +249,6 @@ component("node_lib") {
     deps += [ "deps/histogram" ]
   }
   frameworks = []
-  cflags_cc = [
-    "-Wno-deprecated-declarations",
-    "-Wno-implicit-fallthrough",
-    "-Wno-return-type",
-    "-Wno-sometimes-uninitialized",
-    "-Wno-string-plus-int",
-    "-Wno-unused-function",
-    "-Wno-unused-label",
-    "-Wno-unused-private-field",
-    "-Wno-unused-variable",
-  ]
 
   if (v8_enable_i18n_support) {
     deps += [ "//third_party/icu" ]
@@ -353,6 +340,5 @@ component("node_lib") {
       "src/node_crypto.cc",
       "src/node_crypto.h",
     ]
-    cflags_cc += [ "-Wno-sign-compare" ]
   }
 }
--- src/third_party/electron_node/deps/histogram/BUILD.gn.old
+++ src/third_party/electron_node/deps/histogram/BUILD.gn
@@ -1,12 +1,6 @@
 config("histogram_config") {
   include_dirs = [ "include" ]
 
-  cflags = [
-    "-Wno-implicit-function-declaration",
-    "-Wno-incompatible-pointer-types",
-    "-Wno-unused-function",
-    "-Wno-atomic-alignment",
-  ]
 }
 
 static_library("histogram") {
--- src/third_party/electron_node/deps/llhttp/BUILD.gn.old
+++ src/third_party/electron_node/deps/llhttp/BUILD.gn
@@ -1,6 +1,5 @@
 config("llhttp_config") {
   include_dirs = [ "include" ]
-  cflags = [ "-Wno-unreachable-code" ]
 }
 
 static_library("llhttp") {
--- src/third_party/electron_node/deps/uv/BUILD.gn.old
+++ src/third_party/electron_node/deps/uv/BUILD.gn
@@ -32,24 +32,6 @@ static_library("uv") {
   # This only has an effect on Windows, where it will cause libuv's symbols to be exported in node.lib
   defines += [ "BUILDING_UV_SHARED=1" ]
 
-  cflags_c = [
-    "-Wno-incompatible-pointer-types",
-    "-Wno-bitwise-op-parentheses",
-    "-Wno-implicit-fallthrough",
-    "-Wno-implicit-function-declaration",
-    "-Wno-missing-braces",
-    "-Wno-sign-compare",
-    "-Wno-sometimes-uninitialized",
-    "-Wno-string-conversion",
-    "-Wno-switch",
-    "-Wno-unused-function",
-    "-Wno-unused-result",
-    "-Wno-unused-variable",
-    "-Wno-unreachable-code",
-    "-Wno-unreachable-code-return",
-    "-Wno-unused-but-set-variable",
-    "-Wno-shadow",
-  ]
 
   libs = []
 
